README.md

```diff
@@ -31,8 +29,21 @@ To learn more about Next.js, take a look at the following resources:
 
 You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
 
-## Deploy on Vercel
+## Environment variables
+
+Create a copy of `.env.sample`, name it `.env.local`, and replace the information as needed. Note that not all variables need to be populated.
+
+- `NEXT_PUBLIC_CHAIN_ID`: use 137 for MATIC (Polygon), 80001 for the Mumbai testnet, or 1337 if you are testing locally with Hardhat
```

Comment:
I'm not sure if the current local configuration (Hardhat 1337) is available, do you know  @yu23ki14?  
(I think I could only use testnet, maybe...)